lib/jablotron.js

```diff
@@ -129,12 +130,12 @@ Jablotron.prototype = {
 
     changeAccessoryState: function (accessory, state, callback) {
         var keyboardOrSegmentKey = accessory.getSegmentKey();
-        if (accessory.getKeyboardKey() != null) {
+        if (JablotronConstants.isJablotronPartiallyArmed(state) && accessory.getKeyboardKey() != null) {
```

Comment:
@F4stFr3ddy This is the most import bit - to meet Jablotron API requirements keyboard_key needs to be passed for partialSet state only. For all the other states (set, unset) plugin needs to pass section_key